packages/core/src/components/Onboarding/index.tsx

```diff
@@ -36,6 +37,14 @@ export function Onboarding({
 
   const [page, setPage] = useState<number>(platform === 'web' ? 1 : 0)
 
+  let names = namer(primaryColor)
+
+  useEffect(() => {
+     names = namer(primaryColor)
```

Comment:
Nit: I think there's an extra space before this line. ` names = ....`

Reply:
Yes, I also think names should be defined here.